BonnAnalysis/ComplexInterpolation.lean

```diff
@@ -25,7 +28,7 @@ variable {α β E E₁ E₂ E₃ : Type*} {m : MeasurableSpace α} {n : Measurab
 /- TODO: split proofs into smaller lemmas to recycle code -/
 
 -- All these names are probably very bad
-lemma Real.rpow_le_rpow_iff_left {M:ℝ} (hM: M>0) (a b : ℝ) : M^a ≤ M^b ↔ ((1 ≤ M ∧ a ≤ b ) ∨ (M ≤ 1 ∧ b ≤ a)) := by{
+lemma Real.pow_le_pow_iff {M:ℝ} (hM: M>0) (a b : ℝ) : M^a ≤ M^b ↔ ((1 ≤ M ∧ a ≤ b ) ∨ (M ≤ 1 ∧ b ≤ a)) := by{
```

Comment:
Please undo this name change. I tried to improve the name in https://github.com/fpvandoorn/BonnAnalysis/commit/3a797ef90cd5905b4ff95cd28bc2c6b242a201b0

Reply:
Thanks! I had seen that, but we also changed a hypothesis in a theorem on Friday and I was waiting to fix the other statements that used that theorem before recommitting. I'm not sure what order I am supposed to do things now. Is it fine if I just change this manually in my local version and recommit within this PR?

Reply:
Yes, that works. If you push to this branch, this PR will be automatically updated.
(If you only want to commit part of your changes, you can do that on the command line using `git add -p <myFile>` and then afterwards `git commit -m "my message"` (without the `-a` flag).